dev/MRTCore/mrt/MrtCore.sln

```diff
@@ -21,18 +21,24 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Windows.Applicati
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MrtCoreUnpackagedTests", "Microsoft.Windows.ApplicationModel.Resources\UnpackagedTests\MrtCoreUnpackagedTests.csproj", "{3C618444-4B80-492E-8972-BFAEF8700A52}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.FrameworkUdk.PackageReference", "..\..\..\eng\PackageReference\FrameworkUdk\Microsoft.FrameworkUdk.PackageReference.csproj", "{4E965899-43FD-4654-9613-15E8EB574E01}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
 		Debug|ARM = Debug|ARM
```

Comment:
who builds any cpu? in build pipeline, only x86/x64/arm64 are built

Reply:
I think this was added when I included the FrameworkUDK csproj

Reply:
that's just your config manager setting. but if build pipeline doesn't build anycpu, this is probably not essential.